<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>647</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Cascadia Mono</family>
    <pointsize>10</pointsize>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Fdog-Kit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lib.qrc">
    <normaloff>:/lib/wicon_46.png</normaloff>:/lib/wicon_46.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow,QWidget#centralWidget {
background-color: rgb(67, 77, 88);
border-radius:6px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(67, 77, 88);
color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_title" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(67, 77, 88);
color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label_icon">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/lib/wicon_46.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_title">
         <property name="font">
          <font>
           <family>OPPOSans B</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Fdog-Kit</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_min">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="lib.qrc">
           <normaloff>:/lib/icon-minus.png</normaloff>:/lib/icon-minus.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_max">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="lib.qrc">
           <normaloff>:/lib/Icon_max2.png</normaloff>
           <activeoff>:/lib/icon-copy.png</activeoff>:/lib/Icon_max2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_close">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="lib.qrc">
           <normaloff>:/lib/icon-close.png</normaloff>:/lib/icon-close.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>17</width>
           <height>17</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_line" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>2</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>2</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="Line" name="line">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(108, 117, 125, 65);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_tool" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolButton_newCreate">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>OPPOSans B</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}</string>
         </property>
         <property name="text">
          <string>新建(&amp;N)</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_tool">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>OPPOSans B</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}</string>
         </property>
         <property name="text">
          <string>工具(&amp;T)</string>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>14</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_setting">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>OPPOSans B</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}</string>
         </property>
         <property name="text">
          <string>设置(&amp;S)</string>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>14</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_about">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>OPPOSans B</family>
           <pointsize>10</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}</string>
         </property>
         <property name="text">
          <string>帮助(&amp;H)</string>
         </property>
         <property name="iconSize">
          <size>
           <width>13</width>
           <height>14</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_line_2" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>2</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>3</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="Line" name="line_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(108, 117, 125, 65);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_body" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(67, 77, 88);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(67, 77, 88);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(67, 77, 88);
color: rgb(255, 255, 255);</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="font">
              <font>
               <family>OPPOSans B</family>
               <pointsize>11</pointsize>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
 QTabWidget::tab-bar {
         left:0px;       
}
	


/*设置TabWidget中tab_1的样式*/
#tab_1.QWidget{
	background-color: rgb(108, 117, 125, 65);
}
 
/*设置TabWidget中tab_2的样式*/
#tab_2.QWidget{
	background-color: rgb(108, 117, 125, 65);
}
 
/*设置TabWidget中QTabBar的样式*/
QTabBar::tab{
    background-color: rgb(108, 117, 125, 65);
	/*font-family:Consolas; */   /*设置tab中的文本的字体*/
	font-size:10pt;
	color:#ced4da;    /*设置tab中的文本的颜色*/
	border-top-left-radius: 5px;    /*设置tab的边框的圆角（左上圆角）*/
	border-top-right-radius: 5px;    /*设置tab的边框的圆角（右上圆角）*/

	padding: 2px;
	margin-right:0px;
	margin-left: 0px;
}
 
/*设置TabWidget中QTabBar的tab被选中时的样式*/
QTabBar::tab:selected{
	background-color: #0B0E11;
}
 
/*设置TabWidget中鼠标悬浮在QTabBar的tab上，但未选中该Tab的样式*/
QTabBar::tab:hover:!selected {
	background-color: rgb(100, 117, 125, 125);
}
 
/*设置TabWidget的边框的样式*/
QTabWidget::pane {
    border: 2px solid rgb(108, 117, 125, 65);
}
 

QTabBar::close-button{ 
	/*border-image: url(:lib/Kafka.png);*/
}
QTabBar::close-button:hover{
	/*border-image: url(:/Resources/image/close_hover.png);*/
}


/*当打开多个tab，右侧出现，点击后，可以向前向后的按钮的样式*/
QTabBar QToolButton {
    border: none;
	color: rgb(255, 206, 6);
    background-color: #0b0e11;
}
 
QTabBar QToolButton:hover {
	background-color: #161a1e; 
}</string>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <property name="tabsClosable">
              <bool>true</bool>
             </property>
             <property name="movable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="lib.qrc"/>
 </resources>
 <connections/>
</ui>
